aten/src/ATen/native/mkldnn/RegisterMkldnnOpContextClass.cpp

```diff
@@ -71,6 +71,10 @@ TORCH_LIBRARY(mkldnn, m) {
       "mkldnn::_reorder_convolution_weight(Tensor self, int[2] padding=0, int[2] stride=1, int[2] dilation=1, int groups=1, int[]? input_size=None) -> Tensor Y"));
   m.def(TORCH_SELECTIVE_SCHEMA(
       "mkldnn::_reorder_mkldnn_rnn_layer_weight(Tensor weight0, Tensor weight1, int hidden_size, bool reverse, bool has_biases, bool batch_first, int[]? input_size=None) -> Tensor[] Y"));
+#if AT_ONEDNN_GRAPH_ENABLED()
+  m.def(TORCH_SELECTIVE_SCHEMA(
+      "mkldnn::_graph_sdpa_pattern(Tensor Query, Tensor Key, Tensor Value, Tensor? scale, Tensor? inverse_scale, Tensor? attn_mask, Tensor? causal_mask, Tensor? causal_mask_value, bool transpose_query=False, bool transpose_key_twice=False, bool transpose_value=False, bool transpose_key_once=False, bool output_requires_transpose_and_reorder=False) -> Tensor Y"));
```

Comment:
how about name it `mkldnn::_spda_fusion`?

Reply:
Sounds better, thanks! :)